src/parsers/texttracks/ttml/html/generate_css_test_outline.ts

```diff
@@ -28,8 +30,16 @@ export default function generateCSSTextOutline(
   color : string,
   thickness : string|number
 ) : string {
-  return `-1px -1px ${thickness} ${color},` +
-         `1px -1px ${thickness} ${color},` +
-         `-1px 1px ${thickness} ${color},` +
-         `1px 1px ${thickness} ${color}`;
+  let thick = thickness;
+  if (isNonEmptyString(thickness) && thickness.trim().endsWith("%")) {
+    // As em and % are basically equivalent in CSS
+    // (they both are relative to the parent)
```

Comment:
From re-reading specs, they both are relative to the font-size of the current element:
https://www.w3.org/TR/css-values-3/#font-relative-lengths
https://www.w3.org/TR/2018/REC-ttml1-20181108/#style-attribute-textOutline